code/game/machinery/vendors/contraband_vendors.dm

```diff
@@ -179,3 +179,61 @@
 
 	armor = list(MELEE = 100, BULLET = 100, LASER = 100, ENERGY = 100, BOMB = 0, RAD = 0, FIRE = 100, ACID = 50)
 	resistance_flags = FIRE_PROOF
+
+/obj/machinery/economy/vending/toyliberationstation/secured
+	name = "\improper Syndicate Donksoft Toy Vendor"
+	desc = "An ages 8 and up approved vendor that dispenses toys. If you were to find the right wires, you can unlock the adult mode setting!"
+	icon_state = "syndi"
+	icon_lightmask = "syndi"
+	slogan_list = list("Get your cool toys today!",
+					"Trigger a valid hunter today!",
+					"Quality toy weapons for cheap prices!",
+					"Give them to HoPs for all access!",
+					"Give them to HoS to get permabrigged!")
+
+	ads_list = list("Feel robust with your toys!",
+					"Express your inner child today!",
+					"Toy weapons don't kill people, but valid hunters do!",
+					"Who needs responsibilities when you have toy weapons?",
+					"Make your next murder FUN!")
+
+	vend_reply = "Come back for more!"
+	category = VENDOR_TYPE_RECREATION
+	products = list(/obj/item/gun/projectile/automatic/toy = 10,
+					/obj/item/gun/projectile/automatic/toy/pistol= 10,
+					/obj/item/gun/projectile/shotgun/toy = 10,
+					/obj/item/toy/sword = 10,
+					/obj/item/ammo_box/foambox = 20,
+					/obj/item/toy/foamblade = 10,
+					/obj/item/toy/syndicateballoon = 10,
+					/obj/item/clothing/suit/syndicatefake = 5,
+					/obj/item/clothing/head/syndicatefake = 5) //OPS IN DORMS oh wait it's just an assistant
+
+	contraband = list(/obj/item/gun/projectile/shotgun/toy/crossbow = 10,   //Congrats, you unlocked the +18 setting!
+					/obj/item/gun/projectile/automatic/c20r/toy/riot = 10,
+					/obj/item/gun/projectile/automatic/l6_saw/toy/riot = 10,
+					/obj/item/gun/projectile/automatic/sniper_rifle/toy = 10,
+					/obj/item/ammo_box/foambox/riot = 20,
+					/obj/item/toy/katana = 10,
+					/obj/item/dualsaber/toy = 5,
+					/obj/item/deck/cards/syndicate = 10) //Gambling and it hurts, making it a +18 item
+
+	armor = list(MELEE = 100, BULLET = 100, LASER = 100, ENERGY = 100, BOMB = 0, RAD = 0, FIRE = 100, ACID = 50)
+	resistance_flags = FIRE_PROOF
```

Comment:
Since this is subtype of `/obj/machinery/economy/vending/toyliberationstation` only define what you want to be changed on this subtype. I dont think you need any of this since you're only changing procs, right?

Reply:
Done in latest!